src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -141,7 +142,7 @@ public Test BuildFrom(IMethodInfo method, Test parentSuite)
                     tests.Add(test);
             }
 
-            return tests.Count > 0
+            return tests.Count > 0 || builders.OfType<TestCaseSourceAttribute>().Any()
```

Comment:
Alright. Both `builders.Count > 0` and `(builders.Count > 0 && !builders.OfType<TheoryAttribute>().Any())` make 134 tests fail with:

>   Expected: <NUnit.Framework.Internal.TestMethod>
>   But was:  <NUnit.Framework.Internal.ParameterizedMethodSuite>